Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,147 @@
     "\n",
-    "# General Recommended Structure for Writing Post_Process File\n",
-    "\n",
-    "Below is the recommended general structure to develop the Post_Process file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
-    "\n",
-    "#### 1- Start with visualization of the residuals for most important variables within the flow field (e.g. continuity, streamwise velocity and etc.). Discuss how user should examine the residuals trends to comment on accuracy and convergence of the flow field's simulation. \n",
-    "#### 2- Explain how to visualize the contours of important normalized flow field variables (e.g. normalized velocity, pressure and etc.) on one or multiple sections of the CFD domain. Present figure(s) of visualized variable contours. \n",
-    "#### 3- Present a detail discussion on details of the presented results and link your discussion to the previously explained physics of the flow field and expected results in Physics section.\n",
-    "#### 4- Validate your simulation results via presenting a detail comparison between CFD simulation results and corresponding experimental or other numerical results. Please have in mind that the goal here is not to sell your simulation results. The goal is to highlight the limitations and capabilities of the CFD simulations and used CFD package and code.\n",
-    "#### 5- Conclude the post-processing and validation with a last conclusive paragraph highlighting the pros and cons of the CFD simulations.\n",
-    "#### 6- Cite the source(s) that you have used for writing the content of your Domain file considering the copy right rules.  Make sure to cite the sources you use or refer to within the body of the text as well as in the end!\n",
-    "\n",
-    "# Handy Commands for Writing the Physics file: \n",
+    "<img src=\"./Images/residuals.png\" width = \"600\">\n",
+    "Fig. 1.\n",
```

Comment:
Please add the plot of residual history. If you can not find your simulation data, run a new simulation for Reynolds of 100 with one specific rotational rate and get that plot from those simulations.